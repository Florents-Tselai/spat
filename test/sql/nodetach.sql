SET spat.db = 'test-spat';

SELECT SPSET('k0', 'v0');
SELECT SPSET('k1', 'v1');
SELECT SPSET('k2', 'v2');

SELECT SPGET('k0');
SELECT SPGET('k1');
SELECT SPGET('k2');

SELECT DEL('k0');
SELECT DEL('k1');
SELECT DEL('k2');

SELECT SPGET('k0');
SELECT SPGET('k1');
SELECT SPGET('k2');
