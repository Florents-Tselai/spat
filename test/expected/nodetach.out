SET spat.db = 'test-spat';
SELECT SPSET('k0', 'v0');
 spset 
-------
 v0
(1 row)

SELECT SPSET('k1', 'v1');
 spset 
-------
 v1
(1 row)

SELECT SPSET('k2', 'v2');
 spset 
-------
 v2
(1 row)

SELECT SPGET('k0');
 spget 
-------
 v0
(1 row)

SELECT SPGET('k1');
 spget 
-------
 v1
(1 row)

SELECT SPGET('k2');
 spget 
-------
 v2
(1 row)

SELECT DEL('k0');
 del 
-----
 t
(1 row)

SELECT DEL('k1');
 del 
-----
 t
(1 row)

SELECT DEL('k2');
 del 
-----
 t
(1 row)

SELECT SPGET('k0');
 spget 
-------
 
(1 row)

SELECT SPGET('k1');
 spget 
-------
 
(1 row)

SELECT SPGET('k2');
 spget 
-------
 
(1 row)

